src/components/molecules/PieChartView.tsx

```diff
@@ -21,7 +21,7 @@ interface IProps {
 
 // hardcoded colors, will be changed
 //TODO CHANGE COLORS
-const COLORS = ['#AE0721', '#818386', '#d90000', '#890505', '#6a6a6a'];
+const COLORS = collisionPieChartColors;
```

Comment:
This is a generic pie chart. It should know nothing about "collision"
rename `collisionPieChartColors` to `pieChartColors`